frontend/src/questions/components/Item.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component, PropTypes } from "react";
+import S from "./List.css";
+
+import Labels from "./Labels.jsx";
+import LabelPopover from "../containers/LabelPopover.jsx";
+
+import Icon from "metabase/components/Icon.jsx";
+import CheckBox from "metabase/components/CheckBox.jsx";
+
+import cx from "classnames";
+import pure from "recompose/pure";
+
+const Item = ({ id, name, created, by, selected, favorite, archived, icon, labels, allLabels, setItemSelected, setFavorited, setArchived }) =>
```

Comment:
I've been pretty bad about it, but you can do `Item.propTypes = { }`. I can go back and do it.